webui/cypress/integration/server-details.spec.js

```diff
@@ -45,13 +45,36 @@ describe('Server details', () => {
     cy.get('.meta-test__ServerDetailsModal button').contains('Storage').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('Network').click();
     cy.get('.meta-test__ServerDetailsModal button').contains('General').click();
+    cy.get('.meta-test__ServerDetailsModal button').contains('Issues 0').click();
+
     cy.get('.meta-test__ServerDetailsModal').closest('div').find('.meta-test__youAreHereIcon');
+
+    cy.get('.meta-test__ServerDetailsModal button:contains(Select zone)').click();
+    cy.get('div').contains('You have no any zone,');
+    cy.get('div').contains('please add one.')
+    cy.get('button:contains(Add new zone)').click();
+    cy.get('.ZoneAddModal input[name="uri"]').type('Narnia');
+    cy.get('.meta-test__ZoneAddSubmitBtn').click();
+    cy.get('.ZoneAddModal').should('not.exist');
+    cy.get('.meta-test__ServerDetailsModal').find('button:contains(Zone Narnia)');
+    cy.get('button:contains(Close)').click();
+    cy.get('.meta-test__ServerDetailsModal').should('not.exist');
+    
+    cy.get('li').contains('dummy-1').closest('li')
+      .find('.meta-test__ReplicasetServerListItem__dropdownBtn').click();
+    cy.get('.meta-test__ReplicasetServerListItem__dropdown *')
+      .contains('Server details').click();
+    cy.get('.meta-test__ServerDetailsModal').find('button:contains(Zone Narnia)').click({force: true});
+    cy.get('div').contains('You have no any zone,').should('not.exist');
+    cy.get('button:contains(Add new zone)').should('be.enabled');
+
     cy.get('.meta-test__ServerDetailsModal button').contains('Close').click();
   });
 
   it('Dead server', () => {
-    cy.task('tarantool', {code: `_G.cluster:server('dummy-2'):stop()`});
-
+    cy.exec('kill -SIGSTOP $(lsof -sTCP:LISTEN -i :8082 -t)', {failOnNonZeroExit: false});
```

Comment:
The behavior of sigstop radically differs from killing the process. You can add it as another test group, but please don't change this one.